test/src/device-request-prompt.spec.ts

```diff
@@ -60,4 +59,46 @@ describe('device request prompt', function () {
       })
     ).rejects.toThrow(TimeoutError);
   });
+
+  it('can be opened', async function () {
+    const {page, httpsServer} = state;
+
+    await page.goto(httpsServer.PREFIX + '/device-request.html');
+
+    await expect(
+      Promise.all([
+        page.waitForDevicePrompt({timeout: 100}),
```

Comment:
The test timeout on CI here and other places where we put it at `100ms` the environment there may too slow to execute them, let remove it everywhere expect where we test that we get a `TimeoutError`.

Reply:
done in https://github.com/puppeteer/puppeteer/pull/11185/commits/e7cadfe88180e6229c6fd315c73e08cebc0f68b5